src/containers/alert.jsx

```diff
@@ -41,9 +45,7 @@ class Alert extends React.Component {
     }
 }
 
-const mapStateToProps = state => ({
-    state: state
```

Comment:
this seems to have been completely useless, so I removed it -- it can't be having any effect, right?

Reply:
I vaguely remember putting this in for some lint-like strange reason, but I forget the details, so if it works for you to remove then sounds good!